WordPressKit/DomainsServiceRemote.swift

```diff
@@ -103,7 +103,7 @@ public class DomainsServiceRemote: ServiceRemoteWordPressComREST {
     }
 
     @objc public func getStates(for countryCode: String,
-                                success: @escaping ([State]) -> Void,
+                                success: @escaping ([WPState]) -> Void,
```

Comment:
Once could argue that this is a breaking change and thus would warrant a bump to `5.0.0` instead of `4.37.0`…